int64 id
int64 object_size

#message file containing 2 int64 discrete values
